src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  display: flex;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  --blueLight-color: #00acdc;
+  --white-color: #fff;
+  --black-color: #000;
+}
+
+.menu__head {
+  background: var(--white-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  background-color: var(--white-color);
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav {
+  width: 55vw;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 0;
+}
+
+.menu_logo {
+  height: 40px;
+  width: 40px;
```

Comment:
Move the logo size to a variable as well

Reply:
done